### PR TITLE
fix: update dependencies

### PR DESCRIPTION
**Pull request by favna**
Resolves #552

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,42 +43,44 @@
 		"build:ts": "rollup -c rollup.config.mjs",
 		"watch": "yarn build -w",
 		"bump": "cliff-jumper",
+		"clean": "rimraf browser.{d.ts,js,mjs} cairo.{d.ts,js,mjs} napi-rs.{d.ts,js,mjs} skia.{d.ts,js,mjs} dist",
```
Comment (kyranet) [inline package.json:46]:
```suggestion
		"clean": "rimraf {browser,cairo,napi-rs,skia}.{d.ts,js,mjs} dist",
```

**Review by kyranet — CHANGES_REQUESTED** [review]

File: src/main.ts.template
Comment (kyranet) [inline src/main.ts.template:1]: Please align the `\`s at the end.

**Review by kyranet — CHANGES_REQUESTED** [review]

File: src/main.ts.template
Comment (favna) [reply src/main.ts.template:1]: Read your Discord DMs

File: package.json (hunk 1 above)
Comment (favna) [reply package.json:46]: Read your Discord DMs

**Review by kyranet — APPROVED** [review]

**Pull request merged**